app.py

```diff
@@ -1,7 +1,9 @@
 from random import Random
 from flask import (Flask, request)
+from flask_cors import CORS
```

Comment:
My python is... ahem... rusty to say the least so not sure if this is the preferred way to do this or not 🤷‍♀️

Reply:
Perfect :grin: (well, I haven't used Python very much in the past 5 years but looks good to me :sweat_smile: )